common/game_rules/zz_hfe_vanilla_replace_game_rules.txt

```diff
@@ -112,6 +112,11 @@ can_spawn_debris = {
 		or = {
 			is_country_type = default
 			is_country_type = hfe_pirate
+			is_country_type = hfe_drone
+			is_country_type = drone
```

Comment:
They produce debris when you kill them. I guess I could remove Drones, since they do that, but I am not sure if that will allow them to drop them after that.

Reply:
This trigger for country and mean only default (and pirate) will get debris.
Drones have no science ships.

Reply:
Yes, but the Default Empires that kill them DO have science ships. When the Harvesting Marvel dies and drops its Debris, you can scan them. This concerns the PRODUCTION of Debris.

Reply:
This trigger do not work like you think.

Reply:
How does it work, then?

Reply:
I'm still waiting on the answer for this one. Please explain it to me when you can.

Reply:
this trigger deside if country can get debries from another country fleet.
Mod drones have no science ship, vanilla drones have no ai.

Reply:
Okay, done